tpot/base.py

```diff
@@ -1184,8 +1203,16 @@ def _mate_operator(self, ind1, ind2):
         for _ in range(self._max_mut_loops):
             ind1_copy, ind2_copy = self._toolbox.clone(ind1),self._toolbox.clone(ind2)
             offspring, offspring2 = cxOnePoint(ind1_copy, ind2_copy)
+
             if str(offspring) not in self.evaluated_individuals_:
                 # We only use the first offspring, so we do not care to check uniqueness of the second.
+
+                # update statistics
+                offspring.statistics['predecessor'] = (str(ind1), str(ind2))
+                offspring.statistics['mutation_count'] = ind1.statistics['mutation_count'] + ind2.statistics[
+                    'mutation_count']
```

Comment:
No need for line break here; we use a lax version of the PEP8 guidelines.

Reply:
Ok. Think this will just be flipping a switch in the IDE.